### PR TITLE
Improved readability of tests

### PR DESCRIPTION
**Pull request by WilliamBergamin**
Improved the readability of newly created test to:

-  facilitate the understanding of http status codes in tests
- ensure that content types and other web standards are maintained

I don't know if we want to implement this for every test in the library, maybe someone can comment on this?

### Category (place an `x` in each of the `[ ]`)

* [x] **bolt** (Bolt for Java)
* [x] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [ ] **slack-api-client** (Slack API Clients)
* [ ] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

## Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to the those rules.

### CODE REVIEW

File: bolt-jakarta-servlet/src/test/java/test_locally/servlet_test/InvalidRequestPatternTest.java [hunk 1]
```diff
@@ -27,6 +27,9 @@ public class InvalidRequestPatternTest {
     private AppConfig appConfig = AppConfig.builder().singleTeamBotToken("xoxb-").signingSecret(secret).build();
     private App app;
     private Servlet webApp;
+    private final String TEXT_PLAIN = MimeTypes.Type.TEXT_PLAIN.getContentTypeField().getValue();
+    private final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
+    private final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```
Comment (seratch) [inline bolt-jakarta-servlet/src/test/java/test_locally/servlet_test/InvalidRequestPatternTest.java:32]: If you use all capital letters for these, you can make them static.
```suggestion
    private static final String TEXT_PLAIN = MimeTypes.Type.TEXT_PLAIN.getContentTypeField().getValue();
    private static final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
    private static final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```

File: bolt-jakarta-servlet/src/test/java/test_locally/servlet_test/SlashCommandTest.java [hunk 2]
```diff
@@ -36,6 +40,9 @@ public class SlashCommandTest {
     private AtomicInteger echoCalls = new AtomicInteger(0);
 
     private Responder echoSenderMock = mock(Responder.class);
+    private final String TEXT_PLAIN = MimeTypes.Type.TEXT_PLAIN.getContentTypeField().getValue();
+    private final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
+    private final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```
Comment (seratch) [inline bolt-jakarta-servlet/src/test/java/test_locally/servlet_test/SlashCommandTest.java:45]:
```suggestion
    private static final String TEXT_PLAIN = MimeTypes.Type.TEXT_PLAIN.getContentTypeField().getValue();
    private static final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
    private static final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```

File: bolt-servlet/src/test/java/test_locally/servlet_test/InvalidRequestPatternTest.java [hunk 3]
```diff
@@ -27,6 +30,9 @@ public class InvalidRequestPatternTest {
     private AppConfig appConfig = AppConfig.builder().singleTeamBotToken("xoxb-").signingSecret(secret).build();
     private App app;
     private Servlet webApp;
+    private final String TEXT_PLAIN = MimeTypes.Type.TEXT_PLAIN.getContentTypeField().getValue();
+    private final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
+    private final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```
Comment (seratch) [inline bolt-servlet/src/test/java/test_locally/servlet_test/InvalidRequestPatternTest.java:35]:
```suggestion
    private static final String TEXT_PLAIN = MimeTypes.Type.TEXT_PLAIN.getContentTypeField().getValue();
    private static final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
    private static final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```

File: bolt-servlet/src/test/java/test_locally/servlet_test/SlashCommandTest.java [hunk 4]
```diff
@@ -36,10 +39,13 @@ public class SlashCommandTest {
     private AtomicInteger echoCalls = new AtomicInteger(0);
 
     private Responder echoSenderMock = mock(Responder.class);
+    private final String TEXT_PLAIN = MimeTypes.Type.TEXT_PLAIN.getContentTypeField().getValue();
+    private final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
+    private final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```
Comment (seratch) [inline bolt-servlet/src/test/java/test_locally/servlet_test/SlashCommandTest.java:44]:
```suggestion
    private static final String TEXT_PLAIN = MimeTypes.Type.TEXT_PLAIN.getContentTypeField().getValue();
    private static final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
    private static final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```

File: bolt/src/test/java/test_locally/app/InvalidPayloadPatternTest.java [hunk 5]
```diff
@@ -31,6 +34,8 @@ public class InvalidPayloadPatternTest {
     AuthTestMockServer server = new AuthTestMockServer();
     SlackConfig config = new SlackConfig();
     Slack slack = Slack.getInstance(config);
+    private final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
+    private final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```
Comment (seratch) [inline bolt/src/test/java/test_locally/app/InvalidPayloadPatternTest.java:38]:
```suggestion
    private static final String APPLICATION_JSON = MimeTypes.Type.APPLICATION_JSON.getContentTypeField().getValue();
    private static final String CONTENT_TYPE = HttpHeader.CONTENT_TYPE.toString();
```

**Review by seratch — CHANGES_REQUESTED** [review]
Thanks for the changes. Can you make the constants static?

**Review by filmaj — APPROVED** [review]

**Review by seratch — APPROVED** [review]

**Pull request merged**